### ISSUE TITLE
About training schedule

### ISSUE THREAD

**Issue by chenz97** [hzxie/RMNet#10]
Thank you for your great work and code! I have a question about the training schedule. In the pre-training schedule, is the training protocol exactly identical to the main training stage? For example, the epoch is also 200 epochs for pre-training? Thank you very much!

**Comment by chenz97** [comment]
Sorry to bother, but I have another question about pre-training. I noticed that you used stuff annotations for the COCO dataset. I'm wondering why you are not using the instance segmentation masks, which seems to be more reasonable since VOS targets are usually things rather than stuffs. Thank you.

**Comment by hzxie** [comment]
Sorry for the late reply.

1. The number of epochs during training is not 200 for pretraining. We stop the pretrain when the loss is no longer decrease.
2. Actually, we are using instance segmentation masks. We give the wrong URL in the README.

**Comment by chenz97** [comment]
Thank you for your reply. Did you start from an schedule of 200 epochs and perform early stopping? Could you give a rough estimation of the number of epochs for pre-training? Thanks!

**Comment by hzxie** [comment]
@chenz97 
Sorry, I don't remember.
Actually, I randomly choose a small number of images from COCO for each epoch.

**Comment by chenz97** [comment]
@hzxie Thank you for your reply. Could you give a rough estimation of the validation performance on DAVIS after the pre-training stage? Thanks.

**Comment by hzxie** [comment]
@chenz97 
Actually, the experiment was done one year ago. Therefore, I can not remember the value clearly.
